doc/async_worker.md

```diff
@@ -109,29 +109,29 @@ calls the callback provided when the AsyncWorker class was created, passing
 in the error as the first parameter.
 
 ```cpp
-virtual void OnError(const Error& e);
+virtual void Napi::AsyncWorker::OnError(const Error& e);
 ```
 
 ### Constructor
 
-Creates a new `AsyncWorker`.
+Creates a new `Napi::AsyncWorker`.
 
 ```cpp
-explicit AsyncWorker(const Function& callback);
+explicit Napi::AsyncWorker(const Function& callback);
 ```
 
 - `[in] callback`: The function which will be called when an asynchronous
 operations ends. The given function is called from the main event loop thread.
 
-Returns an AsyncWork instance which can later be queued for execution by calling
+Returns a`Napi::AsyncWork` instance which can later be queued for execution by calling
 `Queue`.
```

Comment:
nit: &#96;Queue&#96; should be fully namespaced.

Reply:
Done